[weston] Unable to compile

armv7l:
```
/usr/local/bin/ld: /usr/local/lib/libgudev-1.0.so.0: undefined reference to `udev_device_get_devlinks_list_entry@LIBUDEV_183'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:4468: weston] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/usr/local/tmp/crew/4.0.0.tar.gz.dir/weston-4.0.0'
make: *** [Makefile:3570: all] Error 2
weston failed to build: `make V=0 -j4` exited with 2
```

--- a/packages/amtk.rb
+++ b/packages/amtk.rb
@@ -8,6 +8,19 @@ class Amtk < Package
   source_url 'https://github.com/GNOME/amtk/archive/33ec171156ebc49d6dad568e6ba39470edb272e1.zip'
   source_sha256 'fc9bad18bbd5421da52e0548f9e91eda180539bb568d1e7116ca65f4f73f4b67'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/amtk-5.2.0-33ec-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/amtk-5.2.0-33ec-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/amtk-5.2.0-33ec-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/amtk-5.2.0-33ec-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '4605dcc0d4505562e1201ec13471d4afa2ad3be9b1a9685190bd3b9f595cb3cf',
+     armv7l: '4605dcc0d4505562e1201ec13471d4afa2ad3be9b1a9685190bd3b9f595cb3cf',
+       i686: '9a358303f926ef725cf4532df5812304024915af13e90f0bfc12d8f58f77fbad',
+     x86_64: 'f15f58b2e32128bcafffc180fddbce945c70be6bb12241bbf0207cac69a3572e',
+  })
+
   depends_on 'gtk3'
   depends_on 'gobject_introspection' => ':build'
   depends_on 'gtk_doc' => ':build'

--- a/packages/mesa.rb
+++ b/packages/mesa.rb
@@ -8,7 +8,18 @@ class Mesa < Package
   source_url 'https://mesa.freedesktop.org/archive/mesa-20.3.0.tar.xz'
   source_sha256 '2999738e888731531cd62b27519fa37566cc0ea2cd7d4d97f46abaa3e949c630'
 
-
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-20.3.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-20.3.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-20.3.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mesa-20.3.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'cce8cade434faa8ac5aeba8ff9d5c8a141fad2758cbec45b7932f735f7249c3b',
+     armv7l: 'cce8cade434faa8ac5aeba8ff9d5c8a141fad2758cbec45b7932f735f7249c3b',
+       i686: '6a727c773fcf12b7f9755451ee99620d13b7589a62684b370a1392f00006cd8f',
+     x86_64: '0638cf75f48e08d9f86ebcec3f2374212c64ba2a621449a883e7816b85459657',
+  })
 
   depends_on 'llvm' => :build
   depends_on 'elfutils'

--- a/packages/weston.rb
+++ b/packages/weston.rb
@@ -24,12 +24,20 @@ class Weston < Package
     depends_on 'xdg_base'
   end
 
+  binary_url ({
+      i686: 'https://dl.bintray.com/chromebrew/chromebrew/weston-9.0.0-chromeos-i686.tar.xz',
+    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/weston-9.0.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+      i686: '8c685a9d0ae049ed457b32a5513aeb170496fe6c534542d4742092cc62673da4',
+    x86_64: '3f391f699eb4e4da0cbb54865d21cf828850889403ad0e99f1065d382ab97f71',
+  })
 
   def self.build
-    ENV['CFLAGS'] = "-fuse-ld=lld"
-    ENV['CXXFLAGS'] = "-fuse-ld=lld"
+    ENV['CFLAGS'] = '-fuse-ld=lld'
+    ENV['CXXFLAGS'] = '-fuse-ld=lld'
     system "meson #{CREW_MESON_OPTIONS} -Dshell-ivi=false -Dremoting=false -Dbackend-default=wayland -Dbackend-drm=false -Dpipewire=false -Dcolor-management-colord=false -Dcolor-management-lcms=false -Dbackend-rdp=false -Dlauncher-logind=false -Dweston-launch=false -Dsystemd=false -Dxwayland-path=#{CREW_PREFIX}/bin/Xwayland build"
-    system "meson configure build"
+    system 'meson configure build'
     system 'ninja -C build'
     system "cat <<'EOF'> weston.ini
 [core]
@@ -46,8 +54,10 @@ EOF"
   end
   
   def self.postinstall
-    puts "To run weston with xwayland try something like this:".lightred
-    puts "export WAYLAND_DISPLAY=wayland-1".lightred
-    puts "WAYLAND_DISPLAY=wayland-0 weston -Swayland-1 --xwayland".lightred
+    puts
+    puts "To run weston with xwayland try something like this:".lightblue
+    puts "export WAYLAND_DISPLAY=wayland-1".lightblue
+    puts "WAYLAND_DISPLAY=wayland-0 weston -Swayland-1 --xwayland".lightblue
+    puts
   end
 end

